sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,12 +3,18 @@
 ### 4.7.1 (Unreleased)
 
 #### Features Added
+* Added response headers directly to SDK item point operation responses. See [PR 35791](https://github.com/Azure/azure-sdk-for-python/pull/35791).
 * SDK will now retry all ServiceRequestErrors (failing outgoing requests) before failing. Default number of retries is 3. See [PR 36514](https://github.com/Azure/azure-sdk-for-python/pull/36514).
 * Added Retry Policy for Container Recreate in the Python SDK. See [PR 36043](https://github.com/Azure/azure-sdk-for-python/pull/36043)
 * Added option to disable write payload on writes. See [PR 37365](https://github.com/Azure/azure-sdk-for-python/pull/37365)
 * Added get feed ranges API. See [PR 37687](https://github.com/Azure/azure-sdk-for-python/pull/37687)
 * Added feed range support in `query_items_change_feed`. See [PR 37687](https://github.com/Azure/azure-sdk-for-python/pull/37687)
 
+#### Breaking Changes
+* Item-level point operations will now return `CosmosDict` and `CosmosList` response types. 
```

Comment:
since we reverted the query changes, should we also remove the CosmosList response type here? (I thought it is only used for query)